packages/suite/src/hooks/suite/useMessageSystemStaking.ts

```diff
@@ -0,0 +1,39 @@
+import {
+    selectIsFeatureDisabled,
+    Feature,
+    selectFeatureMessageContent,
+} from '@suite-common/message-system';
+import { selectLanguage } from 'src/reducers/suite/suiteReducer';
+import { useSelector } from './useSelector';
+
+export const useMessageSystemStaking = () => {
+    const language = useSelector(selectLanguage);
+
+    const isStakingDisabled = useSelector(state =>
+        selectIsFeatureDisabled(state, Feature.ethStake),
+    );
+    const isUnstakingDisabled = useSelector(state =>
+        selectIsFeatureDisabled(state, Feature.ethStake),
+    );
+    const isClaimingDisabled = useSelector(state =>
+        selectIsFeatureDisabled(state, Feature.ethStake),
+    );
+    const stakingMessageContent = useSelector(state =>
+        selectFeatureMessageContent(state, Feature.ethStake, language),
+    );
+    const unstakingMessageContent = useSelector(state =>
+        selectFeatureMessageContent(state, Feature.ethStake, language),
+    );
+    const claimingMessageContent = useSelector(state =>
+        selectFeatureMessageContent(state, Feature.ethStake, language),
```

Comment:
I see only `Feature.ethStake` used but I would expect to see ethUnstake and ethClaim too

Reply:
🤯 wow, good catch

Reply:
97db1a598f166b6725a391e0f6b5bed5ab1867db